aspnetcore/signalr/configuration.md

```diff
@@ -0,0 +1,128 @@
+---
+title: ASP.NET Core SignalR configuration
+author: rachelappel
+description: Configure ASP.NET Core SignalR Apps
+manager: wpickett
```

Comment:
Remove the following metadata fields:
1. manager
1. ms.prod
1. ms.technology
1. ms.topic

These are defined in *docfx.json* now.